charmhelpers/contrib/network/ufw.py

```diff
@@ -366,12 +366,10 @@ def status():
         (1, {'to': '', 'action':, 'from':, '', ipv6: True, 'comment': ''})
     :rtype: Iterator[Tuple[int, Dict[str, Union[bool, str]]]]
     """
-    if six.PY2:
-        raise RuntimeError('Call to function not supported on Python2')
-    cp = subprocess.run(('ufw', 'status', 'numbered',),
-                        stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
-                        check=True, universal_newlines=True)
-    for line in cp.stdout.splitlines():
+    cp = subprocess.check_output(('ufw', 'status', 'numbered',),
```

Comment:
Why are we losing PY2 guards?

Reply:
this was trying to protect the use of `subprocess.run`.  But honestly, subprocess.run was introduced in python35.  i'm trying make charm-helpers continue to work in py34.  Because six.PY2 is false in py34, this began to fail.  

I felt like using check_output is more broadly available across py2 and py34 as well as py38